addons/waypoint/XEH_PREP.hpp

```diff
@@ -0,0 +1,2 @@
+PREP(fnc_moveVehicleWp);
```

Comment:
This shouldn't work in current state at all.
```suggestion
PREP(moveVehicleWp);
```